scalding-core/src/main/scala/com/twitter/scalding/examples/WordCountJob.scala

```diff
@@ -3,8 +3,9 @@ package com.twitter.scalding.examples
 import com.twitter.scalding._
 
 class WordCountJob(args: Args) extends Job(args) {
-  TextLine(args("input")).read.
-    flatMap('line -> 'word) { line: String => line.split("\\s+") }.
-    groupBy('word) { _.size }.
-    write(Tsv(args("output")))
+  TypedPipe.from(TextLine(args("input")))
+    .flatMap { line: String => line.split("\\s+") }
```

Comment:
you want to drop the type of line? `.flatMap { line => line.split("\\s+") }`

Also, I think `.map(_ -> 1L)` is a bit too compact for an example. Can we do: `.map { word => (word, 1L) }`?